fob413/airtech
# 166563525 Email Setup

# What does this PR do?
- Sets up email in the application

# Description of Task to be completed?
- setup email
- send email on successful flight booking

# How should this be manually tested?
- Start the app by following the readme
- Hit the endpoint with the valid post request object `/api/v1/ticket`
- Check your email for the sent ticket confirmation
- Run the test `tox`

# Any background context you want to provide?
NA

# What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166563525
